lang/typescript/typescript.py

```diff
@@ -284,3 +294,8 @@ def code_public_function(text: str):
 
     #     actions.user.code_insert_function(result, None)
 
+    def code_insert_type_annotation(type: str):
+        actions.insert(f": {type}")
+
+    def code_insert_return_type(type: str):
+        actions.insert(f" => {type}")
```

Comment:
One issue here is that this syntax is for defining return types in function type expressions (eg `(foo: string) => string`), but not how you define return type when  declaring a function (eg `foo(): string`).  I wonder what's the right thing to do here 🤔 cc/ @brxck @AndreasArvidsson

Reply:
I don't have a good solution for this problem. Personally I just have `type` at its own thing and not in combination with arrow or colon.

Reply:
I'm leaning towards using `: ` here, and then people can just use "op lambda type string" for the arrow version. Seems more likely to be what you'd expect